pkg/trace/agent/obfuscate.go

```diff
@@ -101,6 +104,27 @@ func (a *Agent) obfuscateSpan(span *pb.Span) {
 			return
 		}
 		span.Meta[tagElasticBody] = o.ObfuscateElasticSearchString(v)
+	case "system":
+		if span.Name == "command_execution" {
+			v, ok := span.Meta[tagShellCommand]
+			if span.Meta == nil || !ok {
+				v, ok = span.Meta[tagExecCommand]
+				if span.Meta == nil || !ok {
+					return
+				}
+
+				cmdExec, cmdIndices, err := o.ObfuscateExecCommand(v)
+				if err != nil {
+					log.Errorf("Error obfuscating exec command %q: %v", v, err)
```

Comment:
Not sure we want to log as `Error` in the agent, I'm not seeing the other obfuscators doing similar 🤔